protocol-configuration/dependencies.ts

```diff
@@ -928,7 +915,7 @@ const dependencies: DependencyMap = {
     contractDependencies: ['rariPool8MasterOracle']
   },
   rariPool8Dai: {
-    contractDependencies: ['rariPool8Comptroller', 'rariPool8DaiIrm', 'rariPool8DaiPCVDeposit']
```

Comment:
Removing `rariPool8DaiPCVDeposit` because the pool has been hacked and there is bad debt. If in the future, the pools are re-floated and fixed - we'll need to re-add the PCVDeposit to the collaterisation oracle right?

Reply:
yes, assuming PCV will stay there